### PR TITLE
Added create new project functionality

### PR DESCRIPTION
**Pull request by d4vidmc**

### CODE REVIEW

File: src/main/java/org/fundacionjala/pivotal/pages/Project.java [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    @FindBy(css = ".tc-form__input")
+    private WebElement projectNameField;
+
+    @FindBy(css = ".tc-account-selector")
+    private WebElement accountSelector;
+
+    @FindBy(css = ".tc-account-selector__option-list li:nth-of-type(1)")
+    private WebElement accountSelectorOptionone;
+
+    @FindBy(xpath = "//*[@id=\"modal_area\"]/div/div[2]/div/form/footer/button[2]")
```
Comment (carledriss) [inline src/main/java/org/fundacionjala/pivotal/pages/Project.java]: @AWT02/awt02 
this is a wrong locator
We will review this in the class

File: src/main/java/org/fundacionjala/pivotal/pages/Project.java [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    public void clickCreateButton() {
+        action.click(createButton);
+    }
+
+    /**
+     * Create new project by given name.
+     *
+     * @param projectName value
+     */
+    public void createNewProject(final String projectName) {
```
Comment (carledriss) [inline src/main/java/org/fundacionjala/pivotal/pages/Project.java]: @AWT02/awt02 
create project implies to send projectName, account, and privacy.
so, the method should handle required and optional fields.

We will talk about this in the class

Comment (d4vidmc) [reply src/main/java/org/fundacionjala/pivotal/pages/Project.java]: Solved on next commit

File: src/main/java/org/fundacionjala/pivotal/pages/Project.java (hunk 1 above)
Comment (d4vidmc) [reply src/main/java/org/fundacionjala/pivotal/pages/Project.java]: locator has changed

**Pull request merged**